app/services/oneoffs/npq/backfill_trn_to_teacher_profile.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Oneoffs::NPQ
+  class BackfillTrnToTeacherProfile
+    def migrate
+      # locate all TeacherProfiles with empty TRN
+      incorrect_teacher_profiles.in_batches.each_record do |profile|
```

Comment:
Needs updating in line with the name change
```suggestion
      teacher_profiles_without_trns.in_batches.each_record do |profile|
```